Started by upstream project "[8mha:////4A+bQWpgGeLrf62Kvp9R0kchXAV8l6qMSbpRKFffssHgAAAAnR+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzuEgZB/az8JP2UxJJE3aLSvLzUIn0A55yqqsYAAAA=[0mdata-runner" build number [8mha:////4HYnbOK2yJKrYvu9FlmWXzfcrajHsRWmAAd3a/DPgB9AAAAAnB+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLQAygGfpZ+Un6KYklibpFpXl5qUX6ZgB6zjkXxwAAAA==[0m6
originally caused by:
 Started by user [8mha:////4AvbKtqCpWGC4/9vlaAOyQD+y56HOfLOpPpkD7K9YRsAAAAAlx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzWEgZu/dLi1CL9xJTczDwAj6GcLcAAAAA=[0madmin
Running as SYSTEM
Building in workspace /var/root/.jenkins/workspace/data-liuxing
using credential root
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url git@github.com:kirinisland/datapipe.git # timeout=10
Fetching upstream changes from git@github.com:kirinisland/datapipe.git
 > git --version # timeout=10
using GIT_SSH to set credentials 
 > git fetch --tags --force --progress -- git@github.com:kirinisland/datapipe.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision 8862dc0b93faeb24f71cee4ff5c950ac2bc56c56 (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 8862dc0b93faeb24f71cee4ff5c950ac2bc56c56 # timeout=10
Commit message: "update log"
 > git rev-list --no-walk 8862dc0b93faeb24f71cee4ff5c950ac2bc56c56 # timeout=10
[data-liuxing] $ /bin/sh -xe /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/jenkins5794029993972663882.sh
+ set -x
+ date_str=
+ '[' '' = '' ']'
++ date +%Y%m%d
+ date_str=20210418
+ output_dir=/Users/hongliang/media/storage/production_data/original_lyric/20210418
+ mkdir -pv /Users/hongliang/media/storage/production_data/original_lyric/20210418
+ python3 runner_qq.py --method search --query $'\346\265\201\350\241\214' --output-suffix liuxing --dump-raw 1 --dump-seg 0 --proxy
INFO:[DataPipeline]:Start crawling for query: ÊµÅË°å
INFO:[DataPipeline]:Processing playlist page: 0, found 43 playlists
INFO:[DataPipeline]:Processing playlist page: 1, found 46 playlists
INFO:[DataPipeline]:Processing playlist page: 2, found 46 playlists
INFO:[DataPipeline]:Processing playlist page: 3, found 49 playlists
INFO:[DataPipeline]:Processing playlist page: 4, found 47 playlists
INFO:[DataPipeline]:Processing playlist page: 5, found 45 playlists
INFO:[DataPipeline]:Processing playlist page: 6, found 47 playlists
INFO:[DataPipeline]:Processing playlist page: 7, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 8, found 48 playlists
INFO:[DataPipeline]:Processing playlist page: 9, found 49 playlists
INFO:[DataPipeline]:Processing playlist page: 10, found 48 playlists
INFO:[DataPipeline]:Processing playlist page: 11, found 48 playlists
INFO:[DataPipeline]:Processing playlist page: 12, found 48 playlists
INFO:[DataPipeline]:Processing playlist page: 13, found 49 playlists
INFO:[DataPipeline]:Processing playlist page: 14, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 15, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 16, found 48 playlists
INFO:[DataPipeline]:Processing playlist page: 17, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 18, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 19, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 20, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 21, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 22, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 23, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 24, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 25, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 26, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 27, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 28, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 29, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 30, found 49 playlists
INFO:[DataPipeline]:Processing playlist page: 31, found 49 playlists
INFO:[DataPipeline]:Processing playlist page: 32, found 19 playlists
INFO:[DataPipeline]:Processing playlist page: 33, found 17 playlists
INFO:[DataPipeline]:Processing playlist page: 34, found 2 playlists
INFO:[DataPipeline]:Processing playlist page: 35, found 17 playlists
INFO:[DataPipeline]:Processing playlist page: 36, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 37, found 44 playlists
INFO:[DataPipeline]:Processing playlist page: 38, found 21 playlists
INFO:[DataPipeline]:Processing playlist page: 39, found 46 playlists
INFO:[DataPipeline]:Processing playlist page: 40, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 41, found 48 playlists
INFO:[DataPipeline]:Processing playlist page: 42, found 50 playlists
INFO:[DataPipeline]:Processing playlist page: 43, found 41 playlists
INFO:[DataPipeline]:Processing playlist page: 44, found 18 playlists
INFO:[DataPipeline]:Processing playlist page: 45, found 16 playlists
INFO:[DataPipeline]:Processing playlist page: 46, found 38 playlists
INFO:[DataPipeline]:Processing playlist page: 47, found 19 playlists
INFO:[DataPipeline]:Processing playlist page: 48, found 30 playlists
INFO:[DataPipeline]:Processing playlist page: 49, found 49 playlists
INFO:[DataPipeline]:Processing playlist page: 50, found 0 playlists
INFO:[DataPipeline]:Total playlists found about topic: ÊµÅË°å: 2134
INFO:[DataPipeline]:PlayList: [7520334743] ÁàÜÊ¨æÊî∂Ââ≤Êú∫ÔºÅÊó∂‰∏ãÊµÅË°åÁÉ≠Ê≠åÂêàÈõÜ (0/2134) has 520 songs.
--- Logging error ---
Traceback (most recent call last):
  File "/Library/Python/3.7/site-packages/urllib3/connectionpool.py", line 662, in urlopen
    self._prepare_proxy(conn)
  File "/Library/Python/3.7/site-packages/urllib3/connectionpool.py", line 948, in _prepare_proxy
    conn.connect()
  File "/Library/Python/3.7/site-packages/urllib3/connection.py", line 308, in connect
    self._tunnel()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/http/client.py", line 911, in _tunnel
    message.strip()))
OSError: Tunnel connection failed: 406 Proxy Error: server_error Error creating ext request: TUN_ERR: DNS resolution failed(empty_address_list)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Python/3.7/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/Library/Python/3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/Library/Python/3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='c.y.qq.com', port=443): Max retries exceeded with url: /lyric/fcgi-bin/fcg_query_lyric.fcg?g_tk=5381&format=json&inCharset=utf-8&outCharset=utf-8&platform=h5&needNewCode=1&nobase64=1&musicid=212374942&_=1513437581324 (Caused by ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 406 Proxy Error: server_error Error creating ext request: TUN_ERR: DNS resolution failed(empty_address_list)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "runner_qq.py", line 58, in process_playlist
    lyric = api.get_song_lyric(song.id)
  File "/private/var/root/.jenkins/workspace/data-liuxing/dao/qq_api.py", line 43, in get_song_lyric
    raw_text = self.fetch_raw(url)
  File "/private/var/root/.jenkins/workspace/data-liuxing/dao/base_api.py", line 34, in fetch_raw
    verify=False)
  File "/Library/Python/3.7/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/Library/Python/3.7/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Library/Python/3.7/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/Library/Python/3.7/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/Library/Python/3.7/site-packages/requests/adapters.py", line 510, in send
    raise ProxyError(e, request=request)
requests.exceptions.ProxyError: HTTPSConnectionPool(host='c.y.qq.com', port=443): Max retries exceeded with url: /lyric/fcgi-bin/fcg_query_lyric.fcg?g_tk=5381&format=json&inCharset=utf-8&outCharset=utf-8&platform=h5&needNewCode=1&nobase64=1&musicid=212374942&_=1513437581324 (Caused by ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 406 Proxy Error: server_error Error creating ext request: TUN_ERR: DNS resolution failed(empty_address_list)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 1034, in emit
    msg = self.format(record)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 880, in format
    return fmt.format(record)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 619, in format
    record.message = record.getMessage()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 380, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "runner_qq.py", line 158, in <module>
    main()
  File "runner_qq.py", line 145, in main
    args.proxy, logger, args.dump_raw, args.dump_seg)
  File "runner_qq.py", line 68, in process_playlist
    logger.error("Exception in processing songs: ", err)
Message: 'Exception in processing songs: '
Arguments: (ProxyError(MaxRetryError("HTTPSConnectionPool(host='c.y.qq.com', port=443): Max retries exceeded with url: /lyric/fcgi-bin/fcg_query_lyric.fcg?g_tk=5381&format=json&inCharset=utf-8&outCharset=utf-8&platform=h5&needNewCode=1&nobase64=1&musicid=212374942&_=1513437581324 (Caused by ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 406 Proxy Error: server_error Error creating ext request: TUN_ERR: DNS resolution failed(empty_address_list)')))")),)
INFO:[DataPipeline]:PlayList: [7819310768] 2020ÂõΩËØ≠ÊµÅË°å&#32;Top&#32;100 (1/2134) has 100 songs.
INFO:[DataPipeline]:PlayList: [1551071404] ÊµÅË°å&#32;&#124;&#32;ËøëÂπ¥Êù•ÁÅ´ÁàÜÂÖ®ÁΩëÁöÑÊ≠åÊõ≤ (2/2134) has 292 songs.
--- Logging error ---
Traceback (most recent call last):
  File "/Library/Python/3.7/site-packages/urllib3/connectionpool.py", line 662, in urlopen
    self._prepare_proxy(conn)
  File "/Library/Python/3.7/site-packages/urllib3/connectionpool.py", line 948, in _prepare_proxy
    conn.connect()
  File "/Library/Python/3.7/site-packages/urllib3/connection.py", line 360, in connect
    ssl_context=context,
  File "/Library/Python/3.7/site-packages/urllib3/util/ssl_.py", line 370, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/ssl.py", line 412, in wrap_socket
    session=session
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/ssl.py", line 853, in _create
    self.do_handshake()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/ssl.py", line 1117, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1039: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Python/3.7/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/Library/Python/3.7/site-packages/urllib3/connectionpool.py", line 720, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/Library/Python/3.7/site-packages/urllib3/util/retry.py", line 436, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='c.y.qq.com', port=443): Max retries exceeded with url: /lyric/fcgi-bin/fcg_query_lyric.fcg?g_tk=5381&format=json&inCharset=utf-8&outCharset=utf-8&platform=h5&needNewCode=1&nobase64=1&musicid=233583097&_=1513437581324 (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:1039: The handshake operation timed out')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "runner_qq.py", line 58, in process_playlist
    lyric = api.get_song_lyric(song.id)
  File "/private/var/root/.jenkins/workspace/data-liuxing/dao/qq_api.py", line 43, in get_song_lyric
    raw_text = self.fetch_raw(url)
  File "/private/var/root/.jenkins/workspace/data-liuxing/dao/base_api.py", line 34, in fetch_raw
    verify=False)
  File "/Library/Python/3.7/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/Library/Python/3.7/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Library/Python/3.7/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/Library/Python/3.7/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/Library/Python/3.7/site-packages/requests/adapters.py", line 510, in send
    raise ProxyError(e, request=request)
requests.exceptions.ProxyError: HTTPSConnectionPool(host='c.y.qq.com', port=443): Max retries exceeded with url: /lyric/fcgi-bin/fcg_query_lyric.fcg?g_tk=5381&format=json&inCharset=utf-8&outCharset=utf-8&platform=h5&needNewCode=1&nobase64=1&musicid=233583097&_=1513437581324 (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:1039: The handshake operation timed out')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 1034, in emit
    msg = self.format(record)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 880, in format
    return fmt.format(record)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 619, in format
    record.message = record.getMessage()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/logging/__init__.py", line 380, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "runner_qq.py", line 158, in <module>
    main()
  File "runner_qq.py", line 145, in main
    args.proxy, logger, args.dump_raw, args.dump_seg)
  File "runner_qq.py", line 68, in process_playlist
    logger.error("Exception in processing songs: ", err)
Message: 'Exception in processing songs: '
Arguments: (ProxyError(MaxRetryError("HTTPSConnectionPool(host='c.y.qq.com', port=443): Max retries exceeded with url: /lyric/fcgi-bin/fcg_query_lyric.fcg?g_tk=5381&format=json&inCharset=utf-8&outCharset=utf-8&platform=h5&needNewCode=1&nobase64=1&musicid=233583097&_=1513437581324 (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:1039: The handshake operation timed out')))")),)
INFO:[DataPipeline]:PlayList: [7823130341] 2020Ëã±ËØ≠ÊµÅË°å&#32;Top&#32;100 (3/2134) has 100 songs.
INFO:[DataPipeline]:PlayList: [7233029979] Á≤§ËØ≠ÊµÅË°å„Éª90ÂêéÁöÑÁã¨ÂÆ∂ËÆ∞ÂøÜ (4/2134) has 285 songs.
INFO:[DataPipeline]:PlayList: [2067204233] ÂçéËØ≠ÊµÅË°åKTVÂøÖÁÇπÈáëÊõ≤ (5/2134) has 310 songs.
INFO:[DataPipeline]:PlayList: [1790504159] ÈùíÊò•ËÆ∞ÂøÜ&#32;&#124;&#32;90ÂêéÊ†°Âõ≠Â≤ÅÊúàÁöÑÊµÅË°åÊ≠åÊõ≤ (6/2134) has 222 songs.
INFO:[DataPipeline]:PlayList: [7920278723] Ê∏Ø‰πêÊµÅË°åÊ≠åÔºå‚ÄúÁ≤§‚ÄùÂê¨‚ÄúÁ≤§‚ÄùÈíüÊÑè (7/2134) has 186 songs.
INFO:[DataPipeline]:PlayList: [7920945415] ÂçéËØ≠ÊµÅË°å&#32;&#124;&#32;Èô™‰Ω†Â∫¶ËøáÊº´ÈïøÂ≤ÅÊúà (8/2134) has 112 songs.
INFO:[DataPipeline]:PlayList: [7826297110] 2020Èü©ËØ≠ÊµÅË°å&#32;Top&#32;100 (9/2134) has 100 songs.
INFO:[DataPipeline]:Sleep for: 93
INFO:[DataPipeline]:PlayList: [7469197010] Á´ôÂÜÖËØÑËÆ∫1W&#43;ÁöÑÊµÅË°åÊ≠åÊõ≤ (10/2134) has 365 songs.
INFO:[DataPipeline]:PlayList: [7825263613] 2020Êó•ËØ≠ÊµÅË°å&#32;Top&#32;100 (11/2134) has 100 songs.
INFO:[DataPipeline]:PlayList: [7915451843] ÊµÅË°å‰º§ÊÑü&#58;‰ª•Ëá™Â∑±ÂñúÊ¨¢ÊñπÂºèËøá‰∏ÄÁîü (12/2134) has 206 songs.
INFO:[DataPipeline]:PlayList: [7788746563] Ê¨ßÁæéÊµÅË°å‰∏®ÁÇπÁáÉÊ≤âÂØÇÔºåÈáäÊîæÂÖâÂΩ© (13/2134) has 241 songs.
INFO:[DataPipeline]:PlayList: [4216387623] ÂçéËØ≠ÊµÅË°åÈü≥‰πê8090KTVÂøÖÁÇπÈáëÊõ≤ (14/2134) has 326 songs.
INFO:[DataPipeline]:PlayList: [7256920196] Ê¨ßÁæé&#124;&#32;ÊµÅË°åËΩªÊúâÊ∞ß (15/2134) has 40 songs.
INFO:[DataPipeline]:PlayList: [7911807227] ÂçéËØ≠ÊµÅË°å&#58;&#32;‰∏áÁâ©ÊúâÂ£∞&#32;Èü≥‰Ω†ËÄåÊù• (16/2134) has 192 songs.
INFO:[DataPipeline]:PlayList: [7629833961] ÊµÅË°åÂî±ÁâáÂ∫ó&#32;&#58;&#32;Ë¥©ÂçñËá™Â∑±ÁöÑÊµÅË°åÊ≠å (17/2134) has 71 songs.
INFO:[DataPipeline]:PlayList: [7516321215] Èí¢Áê¥‰∏éÊµÅË°å‰πêÁöÑÂîØÁæéÈÇÇÈÄÖ„ÄéÊåÅÊõ¥„Äè (18/2134) has 89 songs.
INFO:[DataPipeline]:PlayList: [7800801073] ÈÇìÁ¥´Ê£ã&#32;&#58;&#32;‰∏ÄÈ∏£ÊÉä‰∫∫ÁöÑÊµÅË°åÂî±Â∞Ü (19/2134) has 50 songs.
INFO:[DataPipeline]:Sleep for: 151
INFO:[DataPipeline]:PlayList: [638180121] Á≤§ËØ≠ÊµÅË°å&#32;&#124;&#32;ÁªèÂÖ∏‰º†ÊâøÂ≤ÅÊúàÂ¶ÇÊ≠å (20/2134) has 341 songs.
INFO:[DataPipeline]:PlayList: [7706179315] Ê¨ßÁæéÊµÅË°åÂêëÔΩúÁà±‰∫éÂâçÂ•è&#32;Ê≠ª‰∫éÈ´òÊΩÆ (21/2134) has 373 songs.
